README.markdown

```diff
@@ -278,8 +278,11 @@ It is possible to specify the useraccount (and password) that the service will r
        <domain>YOURDOMAIN</domain>
        <user>useraccount</user>
        <password>Pa55w0rd</password>
+       <allowserviceaccountlogonright>true</allowserviceaccountlogonright>
```

Comment:
The example uses a confusing casing. All other XML entries are documented to be lowercase, so I propose to follow this notation. It may become important if someone contributes an XSD schema for the config file